### PR TITLE
DE-414 Add salary tables into new GCP project

### PR DESCRIPTION
**Pull request by aqwvinh**
### Ticket ([DE-414](https://potloc.atlassian.net/browse/DE-414))

---
_from `tiguidou` with :heart:_

### CODE REVIEW

**Comment by aqwvinh** [comment]
Do not merge tap-hibob before changing extractor hibob in meltano project by deselecting payroll integration. We first need to deploy meltano with hibob-payroll pointing at GCP finance-raw project

File: meltano.yml [hunk 1]
```diff
@@ -1,4 +1,5 @@
 version: 1
+default_environment: staging
```
Comment (SBurwash) [inline meltano.yml:2]: This should be removed

File: meltano.yml [hunk 2]
```diff
@@ -11,18 +12,36 @@ plugins:
...
+    - name: authorization
       kind: password
     - name: start_date
       value: '2010-01-01T00:00:00Z'
     config:
       start_date: '2010-01-01T00:00:00Z'
-      username: SERVICE-1726
     select:
-    - '*.*'
+    - 'employees.*'
```
Comment (SBurwash) [inline meltano.yml:22]: Don't we want everything? Or is this purely for testing purposes?

**Review by SBurwash — APPROVED** [review]
Very cool, well done :partying_face:

**Pull request merged**